Methane_MarkDown.Rmd

```diff
@@ -111,3 +109,46 @@ ggplot(data= results, aes(x= year, y= value, color= units))+
          color = "Units") + facet_wrap(~variable)
 ```
 ... to show our data side-by-side.
+
+## Sensitivity Analysis
+
+To examine SSP245's Natural methane  sensitivity we first ask hector to fetch the data. We then create a function to run our parameter set to a particular value. In this case we set our value at `358.05`- twenty percent higher than the original value.
+
+```{r run-with-parameters}
+core <- newcore(ini_file)
+run(core)
+paramr<- fetchvars(core, NA, NATURAL_CH4())
+param<- function(core, parameter, value) {
```

Comment:
Maybe add a comment here quickly explaining what this function does (same for line 136 below).